### DISCUSSION TITLE
How to remove word added to the "spellcheck" list by mistake

### DISCUSSION THREAD

**Discussion by MorganeLeCaignec** [TriliumNext/Trilium#1696] [Q&A] [answered]
I've mistakenly added a wrong spelling of a word to the list of accepted words (with right-click), where can I find the option or where in the DB might I fix this?

**Comment by zadam** [comment, marked as answer]
Not possible from the app. I guess the only way would be to delete the custom dictionary files (but I don't know where these are stored...)

**Reply by 9812ga98sd** [reply]
I need to remove a word as well. Custom dictionary files for???? CK editor 5??? Which dictionary files?

I added the wrong word in trilium.

**Reply by 9812ga98sd** [reply]
C:\Users\USERNAME\AppData\Roaming\Trilium Notes\Custom Dictionary.txt 
all words are there for add to dictionary.

**Reply by MorganeLeCaignec** [reply]
Oh nice! I forgot about this, thanks for finding the file.
For the record, on linux it's in /home/[user name]/.config/Trilium Notes/Custom Dictionary.txt

**Reply by 9812ga98sd** [reply]
Hmm this doesnt actually delete the added words as they are still marked as correct even though deleted... i have no idea how to correct this...
